CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -4,6 +4,11 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 
 <hr>
 
+## v3.4.27 (tbd)
+
+### etcd server
+- Fix [etcdserver: fix corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16047)
```

Comment:
Nit - Should we remove the duplication given this is already under etcd server heading of and sentence starts with `Fix`?

```suggestion
- Fix [corruption check when server has just been compacted](https://github.com/etcd-io/etcd/pull/16047)
```